### PR TITLE
bug ANYRUN - images not displaying

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/45968)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py [hunk 1]
```diff
@@ -128,3 +134,30 @@ def test_taskid_from_url(self):
...
+    """
+    Given:
+        A mock response from the API
+    When:
+        Calling images_from_report
+    Then:
+        Validate that the http request gets called with authorization (headers)
+    """
+    response = {'data': {'analysis': {'content': {'screenshots': [{'permanentUrl': 'www.test.com'}]}}}}
+    mocker.patch('requests.request', side_effect=http_mock)
```
Comment (jochman) [inline Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py:159]: you can use `requests_mock`. see `docs/tutorial-integration/YodaSpeak/Integrations/YodaSpeak/YodaSpeak_test.py:35`

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py [hunk 2]
```diff
@@ -128,3 +134,30 @@ def test_taskid_from_url(self):
...
+    try:
+        images_from_report(response)
+    except ValueError as e:
```
Comment (jochman) [inline Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py:162]: what is the test here? you want to validate that you have or don't have headers?
if you don't expect the function to raise an error, remove the except.

If you want the function to raise an error, use `with pytest.raises`. See `docs/tutorial-integration/YodaSpeak/Integrations/YodaSpeak/YodaSpeak_test.py:64`

**Review by jochman — CHANGES_REQUESTED** [review]
See the question on the test

File: Packs/ANYRUN/ReleaseNotes/1_0_8.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ANY.RUN
+- Fixed an issue where ***anyrun-get-report**** would not display screenshots from the API.
```
Comment (amshamah419) [inline Packs/ANYRUN/ReleaseNotes/1_0_8.md:4]:
```suggestion
- Fixed an issue where the  ***anyrun-get-report*** command would not display screenshots from the API.
```

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py (hunk 1 above)
Comment (MLainer1) [reply Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py:159]: At first I used requests_mock, but I did not found how to check if the request had "headers" sent with it, so I changed back to mocker.patch

Comment (jochman) [reply Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py:159]: https://requests-mock.readthedocs.io/en/latest/matching.html#request-headers

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py [hunk 4]
```diff
@@ -128,3 +132,19 @@ def test_taskid_from_url(self):
...
+    """
+    Given:
+        A mock response from the API
+    When:
+        Calling images_from_report
+    Then:
+        Validate that the http request gets called with authorization (headers)
+    """
+    response = {'data': {'analysis': {'content': {'screenshots': [{'permanentUrl': ' http://www.test.com'}]}}}}
+    requests_mock.get('http://www.test.com')
```
Comment (jochman) [inline Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py:147]:
```suggestion
    requests_mock.get('http://www.test.com', headers={'expected-headers'})
```

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py [hunk 5]
```diff
@@ -128,3 +132,19 @@ def test_taskid_from_url(self):
...
+    images_from_report(response)
+
+    assert requests_mock._adapter.last_request._request.headers["Authorization"] is not None
```
Comment (jochman) [inline Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py:150]:
```suggestion
```

**Review by jochman — APPROVED** [review]
🧙🏿‍♂️

**Comment by MLainer1** [comment]
This pt needs a force merge due to a missing instance of ANY.RUN .

**Pull request merged**